src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/MethodCallHandler.java

```diff
@@ -57,7 +57,8 @@ protected IndentLevel getIndentImpl() {
             // we should increase indentation only if this is the first
             // chained method call which was moved to the next line
             else {
-                indentLevel = new IndentLevel(container.getIndent(), getBasicOffset());
+                indentLevel = new IndentLevel(container.getIndent(),
+                    getIndentCheck().getLineWrappingIndentation());
```

Comment:
@romani 
> basicOffset | Specify how far new indentation level should be indented when on the next line.
> lineWrappingIndentation | Specify how far continuation line should be indented when line-wrapping is present.

I, and the documentation, need some clarification what is the difference between basic offset and line wrapping. Both seem to specify it is the indentation used when the code goes to the next line. I am looking to the documentation to clarify if this change is right or not.

Reply:
In my mind differences are by statements.
if next line for same statement - it is line wrap.
if next line for different statement (child block) - basic offset.

there could possibilities of child block in line wrapped statements.
```
field.doSomething()
    .doSomethig() // line wrap (+4)
    .doSomething( c -> {  // line wrap (+4)
      return c.do();  // basic offset (+2)
    });
```
It is better to recheck such case with IDEs on how they see this situation.

Reply:
@romani Should we add that explanation to the documentation as part of this PR so it is clear?

Reply:
yes, our documentation for most complicated Check is very brief  - https://checkstyle.org/config_misc.html#Indentation

@alinkov , please do us a favor and update documentation a bit to clarify terminology and add example of config/code with line wrap. If you are not sure on how to update doc we can postpone this to separate issue.

Reply:
Documentation is extended as requested at https://github.com/checkstyle/checkstyle/pull/7466
@rnveach , please review, please resolve this conversation if intention of new PR is what you would like to have.